services/broker/Dockerfile

```diff
@@ -1,2 +1,4 @@
 ARG IMAGE_REPO
 FROM ${IMAGE_REPO:-lagoon}/rabbitmq-cluster
+ENV RABBITMQ_DEFAULT_HA_PATTERN='^lagoon-'\
+    RABBITMQ_DEFAULT_VHOST='/'
```

Comment:
The RABBITMQ_DEFAULT_VHOST is already set here - do we need to re-set it https://github.com/amazeeio/lagoon/pull/1226/files#diff-38a0607e913442c3f99084261e69cffdR9 ?

Reply:
Going to remove from there because, if you define a VHOST here different from `/`, you need also to change configuration of consumers.